### PR TITLE
Pr test

### PR DESCRIPTION
**Pull request by EliaMG**
# Calculator
Congratulations! You're submitting your assignment.

## Comprehension Questions

Submitting an open pull request for testing

| Question | Answer |
| --- | --- |
| What went well in your code style, such as indentation, spacing, variable names, readability, etc.? What was lacking? | 
| How did your code keep track of user input?	| 
| How did your code determine what operation to perform? | 
| What opportunities exist to create small methods for this project? | 
| In the next project, what would you change about your process? What would you keep doing? |

### CODE REVIEW

File: calculator.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+elsif operation == "multiply" || operation == "*" || operation == "multiplication"
+    puts "Great, multiplication!"
+    operation = "multi"
+elsif operation == "divide" || operation == "/" || operation == "division"
+    puts "Great, division!"
+    operation = "div"
+elsif operation == "exponent" || operation == "^" || operation == "power"
+    puts "Great, exponents!"
+    operation = "exp"
+elsif operation == "modulus" || operation == "%" || operation == "mod"
```
Comment (EliaMG) [inline calculator.rb:24]: this is a lot of nested elsifs :)

Comment (sallyamoore) [reply calculator.rb:24]: ❤️ `elsif`

**Review by sallyamoore — APPROVED** [review]
Nice job! 🥇

**Comment by sallyamoore** [comment]
this is good

**Review by sallyamoore — COMMENTED** [review]
🥇

**Comment by ShannonLCapper** [comment]
Testing submission

**Comment by ShannonLCapper** [comment]
Testing submission again!

**Comment by ShannonLCapper** [comment]
Another test

**Comment by ShannonLCapper** [comment]
This is another test

**Comment by ShannonLCapper** [comment]
This is a test

**Comment by ShannonLCapper** [comment]
Testing!

**Comment by ShannonLCapper** [comment]
Testing again

**Comment by ShannonLCapper** [comment]
Testing!

**Pull request closed without merging**